ui/StatusQ/src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -356,8 +359,14 @@ Item {
                         font.family: Theme.palette.baseFont.name
                         color: root.enabled ? Theme.palette.directColor1 : Theme.palette.baseColor1 
                         wrapMode: root.multiline ? Text.WrapAtWordBoundaryOrAnywhere : TextEdit.NoWrap
-                        Keys.onReturnPressed: event.accepted = !multiline && !acceptReturn
-                        Keys.onEnterPressed: event.accepted = !multiline && !acceptReturn
+                        Keys.onReturnPressed: {
```

Comment:
Same here, `onAccepted`

Reply:
Will it be emmited here if the keyboard event is not accepted? Enter and Return events are filtered by this code.

Reply:
It's a `TextEdit` and it doesn't have `accepted()`

Reply:
Ah, got it, too bad :/